config/paths.js

```diff
@@ -10,12 +10,12 @@ module.exports = {
   root: rootPath,
   src: join(rootPath, 'src'),
 
-  // Build: Release distribution
-  dist: join(rootPath, 'dist'),
-
   // Build: Package for npm publish
   package: join(rootPath, 'package'),
 
+  // Build: Release distribution
```

Comment:
Again, slightly confusing wording here, since there's a `package/dist` folder as well.

Reply:
I've removed the word "distribution" to help clarify things